example/pubspec.yaml

```diff
@@ -12,6 +12,7 @@ dependencies:
     path: ../
   flutter:
     sdk: flutter
+  badges:
```

Comment:
Because the built-in Material Badge is simple and has no animation 😄 so badges package comes in really handy, they have much more customization, and handle appear/disappear animations of badges implicitly.

Plus, it's been showcased on [Package Of The Week](https://youtu.be/_CIHLJHVoN8) by Flutter themselves, so there's no reason to avoid using it, I think.

Reply:
Okay sounds good!